src/types/gecko-profile.js

```diff
@@ -21,12 +21,31 @@ import type { Milliseconds, Nanoseconds } from './units';
 export type IndexIntoGeckoFrameTable = number;
 export type IndexIntoGeckoStackTable = number;
 
+// Taken from the C++
+//
+// enum class MarkerPhase : int {
+//   Instant = 0,
+//   Interval = 1,
+//   IntervalStart = 2,
+//   IntervalEnd = 3,
+// };
```

Comment:
nit: it would be good to explain more in the comment which enum variant has which time fields.
Or/and we can add this info as `startTime`/`endTime` field comment on `RawMarkerTable`.
(At the first look, I got confused about the use of `interval` since we have `intervalStart/intervalEnd` already. Looking at the marker spec made it clear for me.)

Reply:
Ah yeah, this comment is bad. I guess I fixed it up in a commit that's coming up:

```
// These integral values are exported in the JSON of the profile, and are in the
// RawMarkerTable. They represent a C++ class in Gecko that defines the type of
// marker it is. These markers are then combined together to form the Marker[] type.
// See deriveMarkersFromRawMarkerTable for more information. Also see the constants.js
// file for JS values that can be used to refer to the different phases.
//
// From the C++:
//
// enum class MarkerPhase : int {
//   Instant = 0,
//   Interval = 1,
//   IntervalStart = 2,
//   IntervalEnd = 3,
// };
```